### PR TITLE
Move all data to var

### PR DESCRIPTION
**Pull request by gabriel-v**
move app data files to `/var/lib/liquid`

### CODE REVIEW

File: ansible/roles/common/tasks/main.yml [hunk 1]
```diff
@@ -135,11 +136,35 @@
...
+    dest: /etc/postgresql/9.5/main/postgresql.conf
+
+- name: Move postgresql data
+  shell: mv /var/lib/postgresql/9.5 /var/lib/liquid/data/postgresql/
+
+- name: Move rabbitmq data
+  shell: mv /var/lib/rabbitmq /var/lib/liquid/data/
+
+- name: Link rabbitmq data to new location
+  shell: ln -s /var/lib/liquid/data/rabbitmq /var/lib/rabbitmq
```
Comment (mgax) [inline ansible/roles/common/tasks/main.yml]: This task can't be rerun. Maybe use `creates: /var/lib/rabbitmq`? Same for the two tasks above.

File: ansible/roles/hoover/tasks/main.yml [hunk 2]
```diff
@@ -94,20 +94,37 @@
...
 
-- name: Change ownership of elasticsearch data and logs folders
+- name: Create hoover data and log dirs under /var/lib/liquid/hoover
   file:
-    path: "/opt/hoover/elasticsearch-2.4.3"
+    path: "/var/lib/liquid/{{item}}"
     owner: liquid-apps
     group: liquid-apps
     state: directory
-    recurse: true
```
Comment (mgax) [inline ansible/roles/hoover/tasks/main.yml]: No more recursive, why?

File: ansible/roles/common/tasks/main.yml (hunk 1 above)
Comment (gabriel-v) [reply ansible/roles/common/tasks/main.yml]: will do

File: ansible/roles/hoover/tasks/main.yml (hunk 2 above)
Comment (gabriel-v) [reply ansible/roles/hoover/tasks/main.yml]: this does something else now; that segment of code moved to line 96

**Comment by gabriel-v** [comment]
Left to do: patch `/opt/common/libexec/boot` to check if `/var/lib/liquid` is an empty dir. If it is, then copy all contents of `/opt/var-lib-liquid-template` to it. If not, just log something.

**Pull request merged**